### ISSUE TITLE
Changed default branch to main

### ISSUE THREAD

**Issue by illume** [pygame/pygame#2257]
The default branch is now `main` on pygame.

Unfortunately these open PRs were set to merge against master, and when it changed they were closed by github. 
So they will need to be opened again.

https://github.com/pygame/pygame/pull/2249
https://github.com/pygame/pygame/pull/2226
https://github.com/pygame/pygame/pull/2225
https://github.com/pygame/pygame/pull/2188
https://github.com/pygame/pygame/pull/2162
https://github.com/pygame/pygame/pull/2143
https://github.com/pygame/pygame/pull/2050

This was the main guide I used to do the rename: https://stevenmortimer.com/5-steps-to-change-github-default-branch-from-master-to-main/

Some testing I did on a laptop with a git repo.
- I did a git pull on the master branch, and it pulled down the origin/main branch.
- I made a new branch, and a new PR... and it seems to have made it against main.
- I think... (but not 100% sure) that for existing git repos it is useful to do: `git symbolic-ref refs/remotes/origin/HEAD refs/remotes/origin/main` to update the repo on which branch is the default one. But it doesn't seem to be necessary to make new branches.
- to see default branch in your git repo do: `git symbolic-ref --short refs/remotes/origin/HEAD` it should show "`origin/main`".

Here is the github guide and status of default branch renaming: https://github.com/github/renaming

**Comment by illume** [comment]
Some testing I did on a laptop with a git repo.
- I did a git pull on the master branch, and it pulled down the origin/main branch.
- I made a new branch, and a new PR... and it seems to have made it against main.
- I think... (but not 100% sure) that for existing git repos it is useful to do: `git symbolic-ref refs/remotes/origin/HEAD refs/remotes/origin/main` to update the repo on which branch is the default one. But it doesn't seem to be necessary to make new branches.
- to see default branch in your git repo do: `git symbolic-ref --short refs/remotes/origin/HEAD` it should show "`origin/main`".
